src/views/product-integrations-landing/components/loading-skeleton/integrations-loading-skeleton.module.css

```diff
@@ -0,0 +1,76 @@
+.root {
+	margin-top: 32px;
+}
+
+.placeholder {
+	--loading-skeleton-animation-duration: 4.5s;
+
+	animation: loading var(--loading-skeleton-animation-duration) infinite;
+	background-color: var(--token-color-palette-alpha-100);
+	background-image: linear-gradient(
+		90deg,
+		transparent,
+		var(--token-color-palette-alpha-100),
+		var(--token-color-palette-alpha-200),
+		transparent
+	);
+	background-position: -300% 0;
+	background-repeat: no-repeat;
+	background-size: 75%;
+	border-radius: 2px;
+	opacity: 0.55;
+	width: 100%;
+
+	/* Only enable faster animation if query is supported and value is no-preference */
+	@media (prefers-reduced-motion: no-preference) {
+		--loading-skeleton-animation-duration: 2.5s;
+	}
+}
```

Comment:
In the future we might be able to abstract this to a `SkeletonPart` component or something as part of our future abstraction ideas.  Just a thought!

That being said, I like this straightforward approach for now!

Reply:
There are definitely plans to lift this up into a reusable component! I'd like us to wait to do that until we go through an RFC process, which should start in the next 1-2 weeks. 😊 

- [Asana task](https://app.asana.com/0/1202097197789424/1202791375540720/f)
- [In-progress RFC](https://docs.google.com/document/d/18b-n8c-mWXH_YF05-zn9tJcW9voO-R8RcWH-HCDwCXw/edit)